Assets/LeapMotionModules/ElementRenderer/Scripts/Features/BlendShape/Editor/LeapGuiBlendShapeDataEditor.cs

```diff
@@ -0,0 +1,19 @@
+﻿using UnityEditor;
+using Leap.Unity;
+
+[CanEditMultipleObjects]
+[CustomEditor(typeof(LeapGuiBlendShapeData))]
+public class LeapGuiBlendShapeDataEditor : CustomEditorBase {
+  protected override void OnEnable() {
+    base.OnEnable();
+
+    if (target == null) return;
+
+    dontShowScriptField();
```

Comment:
I also agree that it is annoying, and the symptom is usually a custom editor script that is not properly drawing it.  One of the things CustomEditorBase does is make sure it is drawn correctly.

However, in this case the custom editor is being embeded into the inspector for LeapGuiElement,  and I thought that it would be annoying and visually noise to have a script field for every enabled feature, so I disabled.